solidity/contracts/converter/types/standard-pool/StandardPoolConverter.sol

```diff
@@ -863,14 +895,15 @@ contract StandardPoolConverter is ConverterVersion, IConverter, ContractRegistry
         // destroy the user tokens
         poolToken.destroy(msg.sender, _amount);
 
-        // sync the balances to ensure no mismatch
-        syncReserveBalances(0);
-
         uint256 newPoolTokenSupply = totalSupply.sub(_amount);
 
         uint256[2] memory oldReserveBalances;
         uint256[2] memory newReserveBalances;
-        (oldReserveBalances[0], oldReserveBalances[1]) = reserveBalances();
+
+        // transfer the fees and sync the balances to ensure no mismatch
+        (oldReserveBalances[0], oldReserveBalances[1]) = processNetworkFees(0);
+
+        uint256[] memory reserveAmounts = removeLiquidityReserveAmounts(_amount, totalSupply, oldReserveBalances);
```

Comment:
duplicate line

Reply:
That's actually a "leftover" of the pull from 0.6.34, which yields a compilation error; fixed.